spec/validators/claim/advocate_claim_validator_spec.rb

```diff
@@ -7,7 +7,14 @@
 
   let(:claim) { create(:advocate_claim) }
 
-  before { seed_fee_schemes }
+  before(:all) do
+    seed_fee_schemes
+    seed_fee_types
+  end
+
+  after(:all) do
+    clean_database
+  end
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3536/files#r492591528)

Reply:
👍 have amended to not seed fee_types or use the before/after(:all) blocks

Reply:
👍 have amended to not seed fee_types or use the before/after(:all) blocks